docs/docs/installation.md

```diff
@@ -125,3 +127,17 @@ cd pomerium
 make
 ./bin/pomerium-cli --help
 ```
+
+## Release cycle
+
+The current release cycle is aligned on a monthly basis. Pre-`1.0.0` we target a `MINOR` release on or around the **first day of each month**. We try to hit the targets as closely as possible, while still delivering a quality release.
+
+Pomerium uses [Semantic Versioning](https://semver.org/). In practice this means for a given version number **vMAJOR**.**MINOR**.**PATCH** (e.g. `v0.1.0`):
+
+- **MAJOR** indicates a incompatible API changes,
```

Comment:
`a` should be `an` and `changes` should be `change`, or `a` should be removed entirely